backend/dal/Services/Admin/Concrete/UserService.cs

```diff
@@ -254,6 +250,12 @@ public Paged<AccessRequest> GetAccessRequests(AccessRequestFilter filter)
                 .Include(p => p.User)
                 .AsNoTracking();
 
+            var userAgencies = this.User.GetAgencies();
```

Comment:
so part of this makes sense - a system admin should get all of the access requests regardless of what agency they are in. However, what about agency admins that have no agency? won't this mean that they'll get all access requests as well?

Reply:
Looks like the status must be non-null, in that case you could move the three nullable params to the end and default them to null.

Reply:
You right about the agency admins with no agencies, however, the chances of that happening is very low. @Fosol what do you think?

Reply:
Users shouldn't be able to get past the Access Request Page without an Agency.  Currently the only way to by-pass if if we manually go to Keycloak and add agencies.

Reply:
So it's very low